torch/_inductor/ir.py

```diff
@@ -2883,11 +2883,14 @@ class TemplateBuffer(Buffer):
     that we can fuse an epilogue onto.
     """
 
-    def __init__(self, layout, inputs, make_kernel_render):
+    def __init__(self, layout, inputs, make_kernel_render, name=None):
```

Comment:
In order to be able to clone these (needed for fusion), I needed a way to pass in the name instead of always registering a new one.